ui_src/src/domain/stationsList/stationBoxOverview/index.js

```diff
@@ -56,38 +60,29 @@ const StationBoxOverview = ({ station, handleCheckedClick, isCheck }) => {
         }
     }, []);
 
+    const replaceTimeUnits = (inputString) => {
```

Comment:
You don't need this function; you can change it directly inside the convertor function "convertSecondsToDate."